vignettes/ar-grid.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "Create A&R Grid"
+output:
+  rmarkdown::html_vignette
+resource_files:
+   - package_manual.pdf
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Create A&R Grid}
+---
+
+```{r, echo = FALSE, message = FALSE, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  out.width = "100%"
+)
+```
+
+```{r, message = FALSE}
+library(metalite)
+library(r2rtf)
+library(dplyr)
+```
+
+## Overview
+
+In this document, we illustrate how to create the A&R grid by `metalite`.
+
+## Build a Meta Data
+
+### Step 1: input the population and observation datasets.
```

Comment:
1. input → digest?
2. Avoid the ending `.` in headings
3. Need a blank line between headings and code block below